### ISSUE TITLE
Log window scrolling back to top on its own after update to 11.0.2 

### ISSUE THREAD

**Issue by serge-vk** [nextcloud/server#3644]
### Steps to reproduce
1. Update to the latest stable NextCloud version (11.0.2)
2. Make sure the log is sufficiently long to trigger dynamic loading  
3. Scroll down to the end of the log window

### Expected behaviour
Additional log entries should be loaded below and the window position from the top remain unchanged

### Actual behaviour
Additional log entries are loaded, but each time it happens, the log window is rewound back to the very top (first entry).

### Server configuration
**Operating system**: FreeBSD 11.0 p7

**Web server:** Nginx 1.11.9

**Database:** MariaDB 10.1.21

**PHP version:** 7.1.2

**Nextcloud version:** 11.0.2 (stable)

**Updated from an older Nextcloud/ownCloud or fresh install:**
Updated from ownCloud 9 to NextCloud 10; then updated to 11.0, 11.0.1, and 11.0.2

**Where did you install Nextcloud from:**
NextCloud 10 and 11.0 were downloaded from https://nextcloud.com/install/ and manually installed. Updates to 11.0.1 and 11.0.2 were made through web interface.

**Signing status:**
<details>
<summary>Signing status</summary>

```
No errors have been found.
```
</details>

**List of activated apps:**
<details>
<summary>App list</summary>

```
Enabled:
  - activity: 2.4.1
  - admin_audit: 1.1.0
  - announcementcenter: 3.0.0
  - bookmarks: 0.9.1
  - calendar: 1.5.0
  - comments: 1.1.0
  - contacts: 1.5.3
  - dav: 1.1.1
  - direct_menu: 0.10.0
  - encryption: 1.4.1
  - federatedfilesharing: 1.1.1
  - federation: 1.1.1
  - files: 1.6.1
  - files_accesscontrol: 1.1.2
  - files_automatedtagging: 1.1.1
  - files_external: 1.1.2
  - files_pdfviewer: 1.0.1
  - files_retention: 1.0.1
  - files_sharing: 1.1.1
  - files_texteditor: 2.2
  - files_trashbin: 1.1.0
  - files_versions: 1.4.0
  - files_videoplayer: 1.0.0
  - firstrunwizard: 2.0
  - gallery: 16.0.0
  - logreader: 2.0.0
  - lookup_server_connector: 1.0.0
  - news: 10.1.0
  - nextcloud_announcements: 1.0
  - notes: 2.2.0
  - notifications: 1.0.1
  - password_policy: 1.1.0
  - provisioning_api: 1.1.0
  - serverinfo: 1.1.1
  - sharebymail: 1.0.1
  - survey_client: 0.1.5
  - systemtags: 1.1.3
  - tasks: 0.9.4
  - templateeditor: 0.2
  - theming: 1.1.1
  - twofactor_backupcodes: 1.0.0
  - updatenotification: 1.1.1
  - workflowengine: 1.1.1
Disabled:
  - apporder
  - external
  - user_external
  - user_ldap
  - user_saml
```
</details>

**The content of config/config.php:**
<details>
<summary>Config report</summary>

```
{
    "system": {
        "instanceid": "ocffm874ikqy",
        "passwordsalt": "***REMOVED SENSITIVE VALUE***",
        "secret": "***REMOVED SENSITIVE VALUE***",
        "trusted_domains": [
            "192.168.0.200",
            "***REMOVED SENSITIVE VALUE***",
            "***REMOVED SENSITIVE VALUE***",
            "***REMOVED SENSITIVE VALUE***"
        ],
        "datadirectory": "\/mnt\/ocData",
        "overwrite.cli.url": "https:\/\/192.168.0.200\/owncloud",
        "dbtype": "mysql",
        "version": "11.0.2.7",
        "dbname": "OwnCloud",
        "dbhost": "localhost:\/tmp\/mysql.sock",
        "dbtableprefix": "oc_",
        "dbuser": "***REMOVED SENSITIVE VALUE***",
        "dbpassword": "***REMOVED SENSITIVE VALUE***",
        "installed": true,
        "loglevel": 2,
        "appstore.experimental.enabled": true,
        "mail_smtpmode": "smtp",
        "mail_from_address": "***REMOVED SENSITIVE VALUE***",
        "mail_domain": "***REMOVED SENSITIVE VALUE***",
        "mail_smtpauthtype": "PLAIN",
        "mail_smtpauth": 1,
        "mail_smtphost": "***REMOVED SENSITIVE VALUE***",
        "mail_smtpport": "465",
        "mail_smtpname": "***REMOVED SENSITIVE VALUE***",
        "mail_smtppassword": "***REMOVED SENSITIVE VALUE***",
        "trashbin_retention_obligation": "30, 365",
        "asset-pipeline.enabled": true,
        "logtimezone": "***REMOVED SENSITIVE VALUE***",
        "memcache.locking": "\\OC\\Memcache\\Redis",
        "filelocking.enabled": "true",
        "memcache.local": "\\OC\\Memcache\\Redis",
        "redis": {
            "host": "\/var\/run\/redis\/redis.sock",
            "port": 0,
            "timeout": 0,
            "dbindex": 0
        },
        "logfile": "\/var\/log\/owncloud.log",
        "activity_expire_days": 365,
        "enable_avatars": true,
        "session_lifetime": 10800,
        "preview_libreoffice_path": "\/usr\/local\/bin\/libreoffice",
        "enabledPreviewProviders": [
            "OC\\Preview\\PNG",
            "OC\\Preview\\JPEG",
            "OC\\Preview\\GIF",
            "OC\\Preview\\BMP",
            "OC\\Preview\\XBitmap",
            "OC\\Preview\\MP3",
            "OC\\Preview\\TXT",
            "OC\\Preview\\MarkDown",
            "OC\\Preview\\PDF",
            "OC\\Preview\\OpenDocument",
            "OC\\Preview\\MSOfficeDoc"
        ],
        "debug": true,
        "preview_office_cl_parameters": " --headless --nologo --nofirststartwizard --invisible --norestore --convert-to pdf --outdir ",
        "mail_smtpsecure": "ssl",
        "versions_retention_obligation": "auto, 365",
        "maintenance": false,
        "htaccess.RewriteBase": "\/nextcloud",
        "updater.release.channel": "stable",
        "theme": "",
        "updater.secret": "***REMOVED SENSITIVE VALUE***"
    }
}
```
</details>

**Are you using external storage, if yes which one:** No

**Are you using encryption:** Yes

**Are you using an external user-backend, if yes which one:** No

### Client configuration
**Browser:** FireFox 51.0.1 (64-bit)

**Operating system:** Mac OS X 10.10.5

#### Web server error log
No entries are added when I open the logging window and repeatedly scroll to the bottom, as it is returned automatically to the top.

#### Nextcloud log (data/nextcloud.log)
No entries are added when I open the logging window and repeatedly scroll to the bottom, as it is returned automatically to the top.

#### Browser log
<details>
<summary>Browser log</summary>
I see the entries in the network/web console similar to the following (with response 200 Ok):

```
https://23.16.187.29/nextcloud/apps/logreader/get?offset=50&count=25
https://23.16.187.29/nextcloud/apps/logreader/get?offset=75&count=25
```
</details>

**Comment by jasonbayton** [comment]
I can replicate this behaviour.

**Comment by KB7777** [comment]
The same here.

**Comment by MorrisJobke** [comment]
cc @icewind1991

**Comment by MorrisJobke** [comment]
Fixed in 11.0.3 - see https://github.com/nextcloud/logreader/issues/43
